wadas/domain/utils.py

```diff
@@ -1,8 +1,26 @@
 import datetime
+import logging
+import os
+from logging.handlers import RotatingFileHandler
 
 
 def get_timestamp():
     """Method to prepare timestamp string to apply to images naming"""
 
     timestamp = datetime.datetime.now().strftime("%Y%m%d-%H%M%S")
     return timestamp
+
+
+def initialize_asyncio_logger(handler=None, level=logging.DEBUG):
```

Comment:
As also Cesare commented in the past, logger init methods are really similar, we should plan a refactor to rese as much code as we can. Let's track this in TODOs